jetstream/config.py

```diff
@@ -329,8 +329,9 @@ def resolve(self, spec: "AnalysisSpec") -> List[Summary]:
             else:
                 raise ValueError(f"Statistic {statistic_name} does not exist.")
 
+            stats_params = params.copy()
```

Comment:
let's use deepcopy to make sure that `pt.pop()` at line 338 doesn't cause trouble

```suggestion
            stats_params = copy.deepcopy(params)
```